lib/screens/sharing.dart

```diff
@@ -217,6 +218,16 @@ class SharingScreenState extends State<SharingScreen>
                                           controlChecked = false;
                                         });
                                       }
+                                      if (permList.contains('Append') ||
```

Comment:
Why not convert to lower then compare? Or can a regexp be used with `contains`? Would you expect a READ and APPEND for example?